C/c4Document1.cc

```diff
@@ -338,7 +339,7 @@ static revidBuffer generateDocRevID(C4Slice body, C4Slice parentRevID, bool dele
     }
     return revidBuffer(generation, digest, kDigestType);
 #else
-    error::_throw(FDB_RESULT_CRYPTO_ERROR);
+    error::_throw(error::ForestDB, FDB_RESULT_CRYPTO_ERROR);
```

Comment:
I almost changed it to that too, but then I left it as it was :p.  I'll change this and the other location.